### PR TITLE
Add Gar-T5 topics to prebuilt

### PR DESCRIPTION
**Pull request by ArthurChen189**
@ronakice

### CODE REVIEW

File: src/main/java/io/anserini/collection/C4Collection.java [hunk 1]
```diff
@@ -141,15 +141,18 @@ public static class Document implements SourceDocument {
...
 
     public Document(JsonNode json, String filename, int jsonLoc) {
       this.raw = json.toPrettyString();
-      this.contents = json.get("text").asText();
 
       this.id = String.format("c4-%s-%06d", filename, jsonLoc);
       this.url = json.get("url").asText();
+      this.contents = json.get("text").asText();
+
+      this.contentsURL =  this.contents + "" + this.url;
```
Comment (ronakice) [inline src/main/java/io/anserini/collection/C4Collection.java:156]: Why is this part of this PR?

**Review by ronakice — CHANGES_REQUESTED** [review]
Another change besides the ones mentioned

File: src/main/java/io/anserini/collection/C4Collection.java [hunk 2]
```diff
@@ -141,15 +141,18 @@ public static class Document implements SourceDocument {
...
```
Comment (lintool) [inline src/main/java/io/anserini/collection/C4Collection.java:155]: Can you explain what this is for? I don't understand why you're concatenating contents and URL...

File: src/test/java/io/anserini/search/topicreader/TopicReaderTest.java [hunk 3]
```diff
@@ -37,7 +37,7 @@ public void testIterateThroughAllEnums() {
       String[] pathParts = topic.path.split("/");
       assertEquals(topic.readerClass, TopicReader.getTopicReaderClassByFile(pathParts[1]));
     }
-    assertEquals(307, cnt);
+    assertEquals(315, cnt);
```
Comment (lintool) [inline src/test/java/io/anserini/search/topicreader/TopicReaderTest.java:40]: Can we please get unit tests for each of the topic files?

**Review by lintool — CHANGES_REQUESTED** [review]

File: src/main/java/io/anserini/collection/C4Collection.java (hunk 2 above)
Comment (ArthurChen189) [reply src/main/java/io/anserini/collection/C4Collection.java:155]: In trec-misinfo-21, we found incorporating URLs into contents can improve performance. I've talked to @ronakice about this issue before; theoretically, I should've opened another PR.

Comment (ronakice) [reply src/main/java/io/anserini/collection/C4Collection.java:155]: yup, let's move this from here.

Comment (lintool) [reply src/main/java/io/anserini/collection/C4Collection.java]: @ronakice sorry, what do you mean?

File: src/test/java/io/anserini/search/topicreader/TopicReaderTest.java (hunk 3 above)
Comment (lintool) [reply src/test/java/io/anserini/search/topicreader/TopicReaderTest.java:40]: Yea, below is good.

File: src/main/java/io/anserini/collection/C4Collection.java (hunk 2 above)
Comment (ArthurChen189) [reply src/main/java/io/anserini/collection/C4Collection.java]: I've restored the file. Will open up a new PR if needed.

**Review by ronakice — APPROVED** [review]
LGTM!

**Review by lintool — APPROVED** [review]

**Pull request merged**